package.json

```diff
@@ -11,7 +11,7 @@
     "url": "https://github.com/thejameskyle"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "jest"
```

Comment:
The CI error is related, we can add those plugin templates to `testPathIgnorePatterns `: https://jestjs.io/docs/en/configuration#testpathignorepatterns-arraystring